README.md

```diff
@@ -123,6 +148,8 @@ We recommend creating a [new Instance](https://partners.platformos.com/instances
 
 Once the module is installed and you have completed the [setup](#setup), you will immediately gain access to the new endpoints created by this module. For example, you can navigate to `/users/new` for the registration form or `/sessions/new` for the login form. This section describes all the functionalities provided by this module and includes a roadmap for potential future enhancements.
 
+To provide your own html (for example for styling purposes), you can leverage the fact that all of the endpoints leverage [theme_render_rc](https://documentation.platformos.com/api-reference/liquid/platformos-tags#theme-render-rc) Liquid Tag, which means you can easily provide your own html for each endpoint by creating a partial in the `app` directory, for example `app/views/partials/users/new.liquid` for the registration endpoint. Refer to the Endpoints section for each functionality to see which partial to create for which endpoint.
```

Comment:
```suggestion
To provide your own HTML (for example, for styling purposes), you can leverage the fact that all of the endpoints use the [theme_render_rc](https://documentation.platformos.com/api-reference/liquid/platformos-tags#theme-render-rc) Liquid tag. This means you can easily provide your own HTML for each endpoint by creating a partial in the `app` directory. For example, you can create `app/views/partials/users/new.liquid` for the registration endpoint. Refer to the Endpoints section for each functionality to see which partial to create for which endpoint.
```

Reply:
Where is the Endpoints section, in which doc? Please provide a link for the user.